npm/webpack-dev-server-fresh/src/helpers/vueCliHandler.ts

```diff
@@ -0,0 +1,24 @@
+import type { CreateFinalWebpackConfig } from '../createWebpackDevServer'
+import debugLib from 'debug'
+import type { Configuration } from 'webpack'
+
+type PresetHandler = Omit<CreateFinalWebpackConfig, 'frameworkConfig'>
+
+const debug = debugLib('cypress:webpack-dev-server-fresh:vueCliHandler')
+
+export function vueCliHandler ({ devServerConfig }: PresetHandler): Configuration {
+  try {
+    const config = require.resolve('@vue/cli-service/webpack.config', {
+      paths: [devServerConfig.cypressConfig.projectRoot],
```

Comment:
The `sourceRelativeWebpackModules.ts` has the framework and importPath (though since this is coming from `@vue/cli-service` it would have to be mapped) which can be used to do a direct require. This works as well, what are your thoughts on which system to use? I'm fine with either but going this route would make the work done inside of `sourceRelativeWebpackModules.ts` for frameworks largely redundant.

Reply:
I think using `require.resolve.paths` is nice since it gives better stack traces if the module isn't found: https://github.com/cypress-io/cypress/pull/20956#pullrequestreview-935793708

My understanding is the `sourceRelativeWebpackModules` is more for sourcing the modules we ship in the binary and pointing to the right ones (eg, `webpack-dev-server`, `html-webpack-plugin`, etc).